stdlib/public/core/StaticString.swift

```diff
@@ -259,7 +259,7 @@ public struct StaticString
 
 extension StaticString {
   public var customMirror: Mirror {
-    return Mirror(reflecting: String(self))
+    return Mirror(reflecting: String(describing: self))
```

Comment:
Here, as a strength reduction, we can directly query `.description`, since we are in the same type:

```Swift
return Mirror(reflecting: description)
```